jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/LabeledPoint.java

```diff
@@ -10,7 +10,7 @@
   /** Label of the point */
   public float label;
   /** Weight of this data point */
-  public float weight = 1.0f;
+  private float weight = 1.0f;
```

Comment:
I think leave it as public is just fine, otherwise, you shall refactor all fields as private and provide getter/setter